### PR TITLE
Replace Hotjar with Segment

### PR DESCRIPTION
**Pull request by Aelbannan**
Added segment tracking to docs theme, and make Segment load Hotjar async.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/hugo-docs/95)

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -437,22 +437,6 @@
     integrity="sha256-7Bdg/UdDMmHDPafpQDYZNPCEbnRP7eAvu+2hEoqRCXs=" crossorigin="anonymous"></script>
   <!-- END: Required by runnable -->
 
-  <script>
-    {{ $currentBranch:= getenv "CURRENT_BRANCH" }}
-    {{ $dgraphEndpoint:= getenv "DGRAPH_ENDPOINT" }}
-
-    window.DGRAPH_ENDPOINT = {{ if (eq $dgraphEndpoint "" ) }}"https://play.dgraph.io/query?latency=true"{{ else }} {{ $dgraphEndpoint }} {{ end }};
```
Comment (ghost) [inline layouts/partials/header.html:444]: I don't believe these lines (441-444) had anything to do with hotjar, but instead were used for the runnable shortcode. I don't think this is used anywhere in the docs though, but FYI removing these lines don't line up with the description of this PR.

Comment (Aelbannan) [reply layouts/partials/header.html:444]: Ah, good catch

Comment (danielmai) [reply layouts/partials/header.html:444]: Yup. These lines should be kept. They're for doc version pages and for runnable code.

Example (version): https://dgraph.io/docs/v20.11/dql/
Example (runnable): https://dgraph.io/docs/v20.11/query-language/functions/#term-matching

Comment (Aelbannan) [reply layouts/partials/header.html:444]: Done

**Review by danielmai — APPROVED** [review]

**Pull request merged**